docs/reference/docker_compose_run.yaml

```diff
@@ -124,7 +124,7 @@ options:
 - option: no-TTY
   shorthand: T
   value_type: bool
-  default_value: "false"
+  default_value: "true"
```

Comment:
This looks odd; is this because we're detecting if we're running with a TTY attached? Perhaps there should be no visible default (or the description needs to be updated to reflect that we're automatically detecting)

Reply:
+1 for documenting this in description

Reply:
Doc updated, thank you folks